src/sentry/static/sentry/app/views/performance/transactionVitals/vitalCard.tsx

```diff
@@ -558,6 +558,9 @@ class VitalCard extends React.Component<Props, State> {
       },
     });
 
+    // TODO(tonyx): This conflicts with the types declaration of `MarkLine`
+    // if we add it in the constructor. So we opt to add it here so typescript
+    // doesn't complain.
```

Comment:
There is a new version of echarts that is typescript based. We might want to give it a shot in the new year.